analysis/analysis_utils.py

```diff
@@ -746,7 +744,7 @@ def apply_clipped_sigmaG(self, old_results, search, filter_params, lh_level):
         print("---------------------------------------")
         return keep_idx_results
 
-    def apply_clipped_average(self, old_results, search, filter_params, lh_level):
```

Comment:
Can you also remove these parameters from the function comment (for this function and the others)?